{{cookiecutter.repo_name}}/setup.py

```diff
@@ -103,7 +103,12 @@ def _unavailable(self, e):
         "BSD 3-Clause License": "BSD-3-Clause",
         "MIT license": "MIT",
         "ISC license": "ISC",
-        "Apache Software License 2.0": "Apache-2.0"}[cookiecutter.license]
+        "Apache Software License 2.0": "Apache-2.0",
+        "GNU Lesser General Public License v3 or later (LGPLv3+)": "GNU Lesser General Public License v3 or later (LGPLv3+)",
```

Comment:
This mapping here exists to map the displayed license (what the user sees when cookiecutter is used) to a a spdx complying identifier. See https://spdx.org/licenses/

Reply:
Thanks for the explanation! Fixed in c783640

Reply:
Also, forgot to mention, it needs to be spdx because tidelift and I suspect other systems too use that standard to get license metadata (as opposed to the license classifiers that are already standardized in python, sadly).